src/d_deh.c

```diff
@@ -1285,6 +1285,7 @@ static const deh_flag_t deh_mobjflags_mbf21[] = {
     {"E4M8BOSS",       MF2_E4M8BOSS     }, // E4M8 boss
     {"RIP",            MF2_RIP          }, // projectile rips through targets
     {"FULLVOLSOUNDS",  MF2_FULLVOLSOUNDS}, // full volume see / death sound
+    {"FLIPPABLE",      MF2_FLIPPABLE    }, // [crispy] randomly flip corpse, blood and death animation sprites
```

Comment:
Technically, it's not an MBF21 flag; it's a Woof implementation detail. Perhaps we should add another DEHACKED field called `Woof Bits`.

Reply:
We can create "Cosmetic Bits" and add them to Woof and Crispy. There are also "Retro Bits" from Doom Retro, where this feature originated, but they work differently: https://github.com/bradharding/doomretro/wiki/Information-for-Mappers#DEHACKED-Lumps

I don't like extensions to MBF21 because the purpose of the standard is that it works everywhere.